app/models/server/models/LivechatRooms.js

```diff
@@ -86,6 +88,27 @@ export class LivechatRooms extends Base {
 			unsetData.tags = 1;
 		}
 
+		if (livechatData) {
```

Comment:
I think you should do this on `Livechat.saveRoomInfo`, as well as you do on `Livechat.saveGuest` method.